help/en/html/apps/PanelPro/GettingStarted.shtml

```diff
@@ -406,7 +406,7 @@
 
     <p class="refPanel">Demo panel: GS-3-Panel Sensors.xml</p>
 
-    <h5></h5>
+    <!-- ><h5></h5> -->
```

Comment:
Just out of curiosity, why did you comment out these empty headers and not remove them?

Reply:
Somebody (Dave Sand?) mentioned on jmri-developers that they were placeholders for later work.

Reply:
Actually they are visual separators without content to provide an end to the two major sections.  I did not like having the summary content look like it was part of the previous sub-section.

Reply:
Ah! In that case, commenting them out wasn't the right thing to do. Let me try putting some non-breaking spaces or similar to see if I can make the HTML content check happy. Sorry for the trouble.

Reply:
Maybe something like `<h5 style="text-align: center">- - - - -</h5>`.

Reply:
Better centering:  `<h5 style="text-align: center; padding-left: 0">- - - - -</h5>`

Reply:
Thanks. Added that better centering.